### PR TITLE
[Attribute] Documentation update of Attribute component

### PR DESCRIPTION
**Pull request by NeverResponse**
| Q             | A
| ------------- | ---
| Doc fix? | yes
| New docs? | no
| Fixed tickets | -

### CODE REVIEW

File: components/Attribute/index.rst [hunk 1]
```diff
@@ -2,12 +2,13 @@ Attribute
...
-   :numbered:
+   :maxdepth: 2
 
    installation
-   model
-   implementing_subject_interface
-   summary
+   models
+   interfaces
+   attribute_types
```
Comment (TheMadeleine) [inline components/Attribute/index.rst]: why don't you describe AttributeTypes in the models file?

Comment (NeverResponse) [inline components/Attribute/index.rst]: Because it's not a model.

Comment (TheMadeleine) [inline components/Attribute/index.rst]: oh, now i see ;D It is is in the /Model/ directory, that's what was misleading.

File: components/Attribute/interfaces.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Model Interfaces
+----------------
+
+.. _component_attribute_model_attribute-interface:
+
+AttributeInterface
+~~~~~~~~~~~~~~~~~~
+
+This interface should be implemented by a class
```
Comment (TheMadeleine) [inline components/Attribute/interfaces.rst]: `implemented by models representing an **Attribute**.`

File: components/Attribute/interfaces.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+.. _component_attribute_model_attribute-subject-interface:
+
+AttributeSubjectInterface
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to characterize an object with attributes,
+the object class needs to implement this interface.
+
+It will aks you to implement the management of :ref:`component_attribute_model_attribute-value` models.
```
Comment (TheMadeleine) [inline components/Attribute/interfaces.rst]: *ask

File: components/Attribute/models.rst [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+   Default attribute types are stored in the :ref:`component_attribute_model_attribute-types` class.
+
+.. _component_attribute_model_attribute-value:
+
+AttributeValue
+--------------
+
+This model binds the subject and the attribute,
+it is used to store the value of the attribute for the subject.
+It has following the properties:
```
Comment (TheMadeleine) [inline components/Attribute/models.rst]: `It has the following properties.`

**Comment by pjedrzejewski** [comment]
Thank you Jasiek! :+1:

**Pull request merged**